src/time_evolution/mcsolve.jl

```diff
@@ -242,29 +268,52 @@ end
         output_func::Function=_mcsolve_output_func,
         kwargs...)
 
-Generates the ODEProblem for an ensemble of trajectories of the Monte Carlo wave function
-time evolution of an open quantum system.
+Generates the ODEProblem for an ensemble of trajectories of the Monte Carlo wave function time evolution of an open quantum system.
+
+In each trajectory, given a system Hamiltonian ``\hat{H}`` and a list of collapse (jump) operators ``\{\hat{J}_i\}_i``, the state ``|\psi(t)\rangle`` evolves according to:
+
+```math
+\frac{\partial}{\partial t} |\psi(t)\rangle= -i \hat{H}_{\textrm{eff}} |\psi(t)\rangle,
+```
+
+where
+
+```math
+\hat{H}_{\textrm{eff}} = \hat{H} - \frac{i}{2} \sum_i \hat{J}_i^\dagger \hat{J}_i.
+```
+
+During the time evolution, each quantum jump ``\hat{J}_i`` event happends at randomly determined time ``t^*`` can be described by
```

Comment:
Same here, I would copy the QuTiP definition.

Reply:
Done~